src/sdkBase.ts

```diff
@@ -260,13 +264,9 @@ export abstract class SdkBase {
     return this._identity && !hasExpired(this._identity.refresh_expires);
   }
 
-  private temporarilyUnavailable() {
-    if (!this._identity && this._apiClient?.hasActiveRequests()) return true;
-    if (
-      this._identity &&
-      hasExpired(this._identity.identity_expires) &&
-      !hasExpired(this._identity.refresh_expires)
-    )
+  private temporarilyUnavailable(identity: Identity | OptoutIdentity | null | undefined) {
+    if (!identity && this._apiClient?.hasActiveRequests()) return true;
```

Comment:
Can you just put a comment in here indicating that this means that the identity is expired but refreshable?  Might help the next person.